pywbem/_subscription_manager.py

```diff
@@ -168,6 +169,20 @@
 __all__ = ['WBEMSubscriptionManager']
 
 
+def validate_persistence_type(pt):
+    """ Validate persistence type integer with value 1,2,3 """
+    if pt is None:
+        return
+    if not isinstance(pt, int):
+        raise ValueError(
+            _format("The persistence_type must be an integer. {0} not "
+                    "allowed.", pt))
+    if pt < 2 or pt > 3:
+        raise ValueError(
+            _format("The persistence_type value must be between 1 and "
+                    "3. {0} not allowed.", pt))
```

Comment:
1. The test in the if condition rejects a value of 1, but the error message permits it.
2. It would be good to add to the error message what the values mean.

Reply:
DONE

Reply:
A resolution to this issue is not in the latest version of the code as of 7/5 (commit b7175d3).

Reply:
In the last update to this PR, the change is in.